src/components/album/AlbumsList.js

```diff
@@ -84,7 +92,9 @@ const AlbumsList = () => {
 
   //call the function to make API request for search input value
   const handleSearch = () => {
-    fetchAlbums(searchType, searchTerm, limit);
+    setLastSearchTerm(searchTerm);
+    setCurrentPage(1);
+    fetchAlbums(searchType, searchTerm);
```

Comment:
If we update the `useEffect` on line 66 to be dependent on the `lastSearchTerm` as suggested in above comment, then we can remove this line( 97) as won't need to explicitly call `fetchAlbums` here, it will just happen by triggering that useEffect call.

Reply:
Code fixed according to them recommendations